app/tag-management/all-tags/all-tags.component.css

```diff
@@ -0,0 +1,11 @@
+.page-header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  border-style: none;
+}
+
+/deep/ .md-tab-label {
```

Comment:
That would work if there was something to attach the class to. Tab labels are auto-generated and there is no separate tag/directive for them (`<md-tab>` only refers to tab's content, not label). And Angular's CSS encapsulation prevents you from styling anything unless you apply the styles/classes _directly_ on the element. I tried to access the labels through other means, such as parent/descendant selectors etc, but nothing works.